plugins/ReactNativeStyle/ReactNativeStyle.js

```diff
@@ -40,53 +48,87 @@ class NativeStyler extends React.Component {
 
   constructor(props: Object) {
     super(props);
-    this.state = {style: null};
+    this.state = {style: null, measureLayout: null};
   }
 
   componentWillMount() {
-    this.props.bridge.call('rn-style:get', this.props.id, style => {
-      this.setState({style});
-    });
+    (this:any)._styleGet = this._styleGet.bind(this);
```

Comment:
Just avoid `Property not found` error of Flow, but I should define `_styleGet: () => void;` instead.